src/NodaTime/Calendars/RegularWeekYearRule.cs

```diff
@@ -12,21 +12,63 @@ namespace NodaTime.Calendars
     // Possible future feature: allow the first day of the week to vary too.
 
     /// <summary>
-    /// WeekYearRule implementing the rules from Noda Time 1.x. Basically this is the ISO
-    /// rules, but with a variable "minimum number of days in the first week".
+    /// Implements <see cref="IWeekYearRule"/> for a rule where weeks are regular:
+    /// every week has exactly 7 days, which means that some overlap from one calendar
```

Comment:
I'm not sure "overlap" is the right word here, since the weeks themselves don't overlap.